tests/cucumber/steps/steps.js

```diff
@@ -180,6 +180,24 @@ module.exports = function getSteps(options) {
     return boxRefArray;
   }
 
+  let doRaw = false;
```

Comment:
I'm not sure if I put it on every one, but most of them. Specifically because any step that's along the lines of `and we make a XYZ call` either already has mocking setup and a `do` call will pass, or it is going to return random stuff so we need to use `doRaw` otherwise it will throw an exception.